### PR TITLE
Add ActionsAllowed and ActionsPermissions to Organization

### PR DESCRIPTION
**Pull request by marcelocarlos**
### What

Implements methods to interact with `/orgs/MY-ORG/actions` API:
- https://docs.github.com/en/rest/reference/actions

### Why

This is important to allow setting up github-actions policies at organization level using this library (e.g. enable/disable github-actions, set/edit filters for allowed actions).

### Additional notes 

I'm fairly new to using this library and I hope I covered all that's needed here, but please let me know if there is something missing or incomplete.

### CODE REVIEW

File: github/github.go
Comment (gmlewis) [inline github/github.go:1100]: Per idiomatic Go, we will never want to have a pointer to a slice since a slice is already a by-reference type.
So please remove this new helper method entirely.

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:1]:
```suggestion
// Copyright 2021 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:14]: Please add a blank comment line above line 14 to separate the description from the reference link like you did on line 26 below.

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:13]: Please make all godoc-style comments complete sentences by adding a period to the end:

```suggestion
// ActionsAllowed represents selected actions that are allowed in an organization.
```

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:25]:
```suggestion
// GetActionsAllowed gets the actions that are allowed in an organization.
```

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:45]:
```suggestion
// EditActionsAllowed sets the actions that are allowed in an organization.
```

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:48]:
```suggestion
func (s *OrganizationsService) EditActionsAllowed(ctx context.Context, org string, actionsAllowed ActionsAllowed) (*ActionsAllowed, *Response, error) {
```

File: github/orgs_actions_allowed_test.go
Comment (gmlewis) [inline github/orgs_actions_allowed_test.go:1]:
```suggestion
// Copyright 2021 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_actions_permissions.go
Comment (gmlewis) [inline github/orgs_actions_permissions.go:1]:
```suggestion
// Copyright 2021 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_actions_permissions.go
Comment (gmlewis) [inline github/orgs_actions_permissions.go:14]: Please add a blank comment line above this line.

File: github/orgs_actions_allowed.go
Comment (gmlewis) [inline github/orgs_actions_allowed.go:18]:
```suggestion
	PatternsAllowed    []string `json:"patterns_allowed,omitempty"`
```

File: github/orgs_actions_permissions.go
Comment (gmlewis) [inline github/orgs_actions_permissions.go:25]:
```suggestion
// GetActionsPermissions gets the GitHub Actions permissions policy for repositories and allowed actions in an organization.
```

File: github/orgs_actions_permissions.go
Comment (gmlewis) [inline github/orgs_actions_permissions.go:48]:
```suggestion
func (s *OrganizationsService) EditActionsPermissions(ctx context.Context, org string, actionsPermissions ActionsPermissions) (*ActionsPermissions, *Response, error) {
```

File: github/orgs_actions_permissions_test.go
Comment (gmlewis) [inline github/orgs_actions_permissions_test.go:1]:
```suggestion
// Copyright 2021 The go-github AUTHORS. All rights reserved.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @marcelocarlos !
Just a few issues to address, and then I think we should be ready for a second LGTM before merging.

File: github/orgs_actions_allowed.go
Comment (marcelocarlos) [reply github/orgs_actions_allowed.go:49]: Thanks @gmlewis for the prompt review, much appreciated!
I'm happy to apply this change, just wanted to double-check first. This pattern of passing pointers to the Edit methods has been used in other files of this library (e.g. https://github.com/google/go-github/blob/master/github/orgs.go#L230), is this something we'd like to change anyway?

File: github/orgs_actions_permissions.go
Comment (marcelocarlos) [reply github/orgs_actions_permissions.go:49]: Same as above: 
I'm happy to apply this change, just wanted to double-check first. This pattern of passing pointers to the Edit methods has been used in other files of this library (e.g. https://github.com/google/go-github/blob/master/github/orgs.go#L230), is this something we'd like to change anyway?

**Comment by marcelocarlos** [comment]
Thank you for the review @gmlewis! Just added a couple of quick follow-up questions and addressed most of the comments.

File: github/orgs_actions_allowed.go
Comment (gmlewis) [reply github/orgs_actions_allowed.go:49]: I'm trying to follow the pattern of passing-by-value for required parameters so that a `nil` does not get passed.

In my opinion, we should try to follow this pattern throughout this repo's API, but I have unfortunately not always been consistent in my code reviews.

Therefore, I recommend we go ahead and pass required parameters by value here in this case, but not break the current API in the other cases... and maybe long-term we should have a single cleanup PR that makes the pattern consistent throughout the repo (after we give users of this repo a chance to discuss the pros/cons of breaking the existing API).

Comment (marcelocarlos) [reply github/orgs_actions_allowed.go:49]: Makes sense, I'll update the PR with the changes 👍

**Comment by marcelocarlos** [comment]
Thanks @gmlewis, all comments addressed now 👍

File: github/orgs_actions_permissions.go
Comment (gmlewis) [inline github/orgs_actions_permissions.go:26]: nit: please get rid of the extra space between "gets" and "the".

File: github/orgs_actions_permissions.go
Comment (gmlewis) [inline github/orgs_actions_permissions.go:46]: nit: "sets the permissions policy" instead of "sets permissions policy".

File: github/orgs_actions_permissions.go
Comment (marcelocarlos) [reply github/orgs_actions_permissions.go]: Fixed in 08be7ab

File: github/orgs_actions_permissions.go
Comment (marcelocarlos) [reply github/orgs_actions_permissions.go]: Fixed in 08be7ab

**Review by gmlewis — APPROVED** [review]
Thank you, @marcelocarlos !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
💙

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
